bundles/net.tourbook/src/net/tourbook/ui/views/TourBlogView.java

```diff
@@ -454,7 +464,7 @@ private String buildWeatherSection(String tourWeather, final boolean addSpacer)
          tourWeather = UI.scrambleText(tourWeather);
       }
 
-      sb.append("<div class='title'>" + Messages.tour_editor_section_weather + "</div>" + NL); //$NON-NLS-1$ //$NON-NLS-2$
+      sb.append(buildSectionTitle(Messages.Tour_Blog_Section_Weather));
       sb.append("<p class='description'>" + WEB.convertHTML_LineBreaks(tourWeather) + "</p>" + NL); //$NON-NLS-1$ //$NON-NLS-2$
 
```

Comment:
I'm not very happy with this and other refactorings, I would like to see the html tags and it's attributes, e.g. class when I have to look into the html code

![image](https://github.com/mytourbook/mytourbook/assets/1283445/6f3429b9-2e0b-47d7-9e14-9617184192a8)

Reply:
>I'm not very happy with this and other refactorings

I reverted to the previous code